packages/asc-core/src/components/ContextMenu/ContextMenuItem.ts

```diff
@@ -0,0 +1,31 @@
+import styled from '../../styled-components'
+import { ContextMenu } from './types'
+import Icon from '../Icon'
+import { getColor, getTypography } from '../../utils/themeUtils'
+
+const ContextMenuItem = styled.li<ContextMenu.ContextMenuItemProps>`
+  padding: 0;
+  margin: 0;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 36px;
+  width: 100%;
+  cursor: pointer;
+  font-size: ${({ theme }) => getTypography(theme).fontWeightRegular};
```

Comment:
You're using a value of `font-weight` for the `font-size` style rule.

Reply:
My bad haha!